### ISSUE TITLE
Error installing on wsl2 UbuntuLTS, node v14.15.4 npm 7.7.5

### ISSUE THREAD

**Issue by kitsunekyo** [addaleax/lzma-native#115]
i'm trying to install a package with a dependency on `lzma-native` but it keeps failing with below error.

the dependency graph is `my app > speedtest-net > decompress-tarx > lzma-native`
as of writing this issue, the package.json from decompress-tarx reads `"lzma-native": "^4.0.5"`

I checked the download url

```bash
npm install
```

```bash
npm ERR! code 1
npm ERR! path /home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native
npm ERR! command failed
npm ERR! command sh -c node-pre-gyp install --fallback-to-build && rimraf build
npm ERR! Failed to execute '/home/aspieslechner/.nvm/versions/node/v14.15.4/bin/node /home/aspieslechner/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/binding-v4.0.6-node-v83-linux-x64/lzma_native.node --module_name=lzma_native --module_path=/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/binding-v4.0.6-node-v83-linux-x64 --napi_version=7 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v83' (1)
npm ERR! node-pre-gyp info it worked if it ends with ok
npm ERR! node-pre-gyp info using node-pre-gyp@0.11.0
npm ERR! node-pre-gyp info using node@14.15.4 | linux | x64
npm ERR! node-pre-gyp WARN Using needle for node-pre-gyp https download 
npm ERR! node-pre-gyp info check checked for "/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/binding-v4.0.6-node-v83-linux-x64/lzma_native.node" (not found)
npm ERR! node-pre-gyp http GET https://node-pre-gyp.addaleax.net/lzma-native/lzma_native-v4.0.6-node-v83-linux-x64.tar.gz
npm ERR! node-pre-gyp http 404 https://node-pre-gyp.addaleax.net/lzma-native/lzma_native-v4.0.6-node-v83-linux-x64.tar.gz
npm ERR! node-pre-gyp WARN Tried to download(404): https://node-pre-gyp.addaleax.net/lzma-native/lzma_native-v4.0.6-node-v83-linux-x64.tar.gz 
npm ERR! node-pre-gyp WARN Pre-built binaries not found for lzma-native@4.0.6 and node@14.15.4 (node-v83 ABI, glibc) (falling back to source compile with node-gyp) 
npm ERR! node-pre-gyp http 404 status code downloading tarball https://node-pre-gyp.addaleax.net/lzma-native/lzma_native-v4.0.6-node-v83-linux-x64.tar.gz 
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@7.1.2
npm ERR! gyp info using node@14.15.4 | linux | x64
npm ERR! gyp info ok 
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@7.1.2
npm ERR! gyp info using node@14.15.4 | linux | x64
npm ERR! gyp info find Python using Python version 3.8.10 found at "/usr/bin/python3"
npm ERR! gyp info spawn /usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/home/aspieslechner/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/aspieslechner/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/aspieslechner/.cache/node-gyp/14.15.4/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/home/aspieslechner/.cache/node-gyp/14.15.4',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/home/aspieslechner/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/home/aspieslechner/.cache/node-gyp/14.15.4/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp: Call to 'sh liblzma-config.sh "/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/build" "/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/deps/xz-5.2.3.tar.bz2"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
npm ERR! gyp ERR! configure error 
npm ERR! gyp ERR! stack Error: `gyp` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onCpExit (/home/aspieslechner/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
npm ERR! gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
npm ERR! gyp ERR! System Linux 5.10.16.3-microsoft-standard-WSL2
npm ERR! gyp ERR! command "/home/aspieslechner/.nvm/versions/node/v14.15.4/bin/node" "/home/aspieslechner/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/binding-v4.0.6-node-v83-linux-x64/lzma_native.node" "--module_name=lzma_native" "--module_path=/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/binding-v4.0.6-node-v83-linux-x64" "--napi_version=7" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v83"
npm ERR! gyp ERR! cwd /home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native
npm ERR! gyp ERR! node -v v14.15.4
npm ERR! gyp ERR! node-gyp -v v7.1.2
npm ERR! gyp ERR! not ok 
npm ERR! node-pre-gyp ERR! build error 
npm ERR! node-pre-gyp ERR! stack Error: Failed to execute '/home/aspieslechner/.nvm/versions/node/v14.15.4/bin/node /home/aspieslechner/.nvm/versions/node/v14.15.4/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/binding-v4.0.6-node-v83-linux-x64/lzma_native.node --module_name=lzma_native --module_path=/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native/binding-v4.0.6-node-v83-linux-x64 --napi_version=7 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v83' (1)
npm ERR! node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
npm ERR! node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
npm ERR! node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1048:16)
npm ERR! node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
npm ERR! node-pre-gyp ERR! System Linux 5.10.16.3-microsoft-standard-WSL2
npm ERR! node-pre-gyp ERR! command "/home/aspieslechner/.nvm/versions/node/v14.15.4/bin/node" "/home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
npm ERR! node-pre-gyp ERR! cwd /home/aspieslechner/code/ISP-SpeedTest-Logger/server/node_modules/lzma-native
npm ERR! node-pre-gyp ERR! node -v v14.15.4
npm ERR! node-pre-gyp ERR! node-pre-gyp -v v0.11.0
npm ERR! node-pre-gyp ERR! not ok
```

**System Info**
Windows 10 21H1
WSL2
```bash
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.2 LTS
Release:        20.04
Codename:       focal
```
node v14.15.4
npm 7.7.5

**Comment by kitsunekyo** [comment]
TLDR: 404 on `https://node-pre-gyp.addaleax.net/lzma-native/lzma_native-v4.0.6-node-v83-linux-x64.tar.gz`

**Comment by addaleax** [comment]
decompress-tarxz seems to not be actively maintained: https://github.com/kevva/decompress-tarxz/pull/14

Bumping lzma-native as a dependency should address this, but I don’t have the powers to do so, sorry.
